exercises/wordy/wordy_test.rb

```diff
@@ -3,95 +3,127 @@
     skip
-    assert_equal(-11, WordProblem.new('What is -1 plus -10?').answer)
+    question = 'What is -1 plus -10?'
+    assert_equal(-11, WordProblem.new(question).answer)
   end
 
-  def test_add_more_digits
+  def test_large_addition
     skip
-    assert_equal 45_801, WordProblem.new('What is 123 plus 45678?').answer
+    question = 'What is 123 plus 45678?'
+    assert_equal 45801, WordProblem.new(question).answer
   end
 
-  def test_subtract
+  def test_subtraction
     skip
-    assert_equal 16, WordProblem.new('What is 4 minus -12?').answer
+    question = 'What is 4 minus -12?'
+    assert_equal 16, WordProblem.new(question).answer
   end
 
-  def test_multiply
+  def test_multiplication
     skip
-    assert_equal(-75, WordProblem.new('What is -3 multiplied by 25?').answer)
+    question = 'What is -3 multiplied by 25?'
+    assert_equal(-75, WordProblem.new(question).answer)
   end
 
-  def test_divide
+  def test_division
     skip
-    assert_equal(-11, WordProblem.new('What is 33 divided by -3?').answer)
+    question = 'What is 33 divided by -3?'
+    assert_equal(-11, WordProblem.new(question).answer)
   end
 
-  def test_add_twice
+  def test_multiple_additions
     skip
     question = 'What is 1 plus 1 plus 1?'
     assert_equal 3, WordProblem.new(question).answer
   end
 
-  def test_add_then_subtract
+  def test_addition_and_subtraction
     skip
     question = 'What is 1 plus 5 minus -2?'
     assert_equal 8, WordProblem.new(question).answer
   end
 
-  def test_subtract_twice
+  def test_multiple_subtraction
     skip
     question = 'What is 20 minus 4 minus 13?'
     assert_equal 3, WordProblem.new(question).answer
   end
 
-  def test_subtract_then_add
+  def test_subtraction_then_addition
     skip
     question = 'What is 17 minus 6 plus 3?'
     assert_equal 14, WordProblem.new(question).answer
   end
 
-  def test_multiply_twice
+  def test_multiple_multiplication
     skip
     question = 'What is 2 multiplied by -2 multiplied by 3?'
     assert_equal(-12, WordProblem.new(question).answer)
   end
 
-  def test_add_then_multiply
+  def test_addition_and_multiplication
     skip
     question = 'What is -3 plus 7 multiplied by -2?'
-    message = 'You should ignore order of precedence. -3 + 7 * -2 = -8, not -17'
-    assert_equal(-8, WordProblem.new(question).answer, message)
+    assert_equal(-8, WordProblem.new(question).answer)
   end
 
-  def test_divide_twice
+  def test_multiple_division
     skip
     question = 'What is -12 divided by 2 divided by -3?'
     assert_equal 2, WordProblem.new(question).answer
   end
 
-  def test_too_advanced
+  def test_unknown_operation
     skip
+    question = 'What is 52 cubed?'
     assert_raises ArgumentError do
-      WordProblem.new('What is 53 cubed?').answer
+      WordProblem.new(question).answer
     end
   end
 
-  def test_irrelevant
+  def test_non_math_question
     skip
+    question = 'Who is the President of the United States?'
```

Comment:
This question doesn't start with 'What is' but that's probably an issue for the common metadata.